<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateMdTitleClass</class>
 <widget class="QMainWindow" name="GenerateMdTitleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenerateMdTitle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>一.99(,)类型</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>本节所有标题的级数:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>本节标题:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_5">
                <property name="maximumSize">
                 <size>
                  <width>23</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>点</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>本节共有多少道题:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QComboBox" name="seriesComboBox">
                <property name="maximumSize">
                 <size>
                  <width>136</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertAfterCurrent</enum>
                </property>
                <item>
                 <property name="text">
                  <string>#</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>##</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>###</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>####</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>#####</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>######</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QCheckBox" name="addSelectStrCheckBox">
                <property name="text">
                 <string>末尾添加&quot;选择&quot;字符</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPlainTextEdit" name="titlePlainTextEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="accessibleDescription">
                 <string/>
                </property>
                <property name="undoRedoEnabled">
                 <bool>true</bool>
                </property>
                <property name="plainText">
                 <string>一</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSpinBox" name="questionNumSpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999999999</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="generatePushButton">
             <property name="text">
              <string>生成文本并自动复制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="generateTextBrowser"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>时间类型</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="1" column="4">
            <widget class="QSpinBox" name="questionNumSpinBox2">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QCheckBox" name="addSelectStrCheckBox2">
             <property name="text">
              <string>末尾添加&quot;选择&quot;字符</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPlainTextEdit" name="titlePlainTextEdit2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="accessibleDescription">
              <string/>
             </property>
             <property name="undoRedoEnabled">
              <bool>true</bool>
             </property>
             <property name="plainText">
              <string>0h59min</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QComboBox" name="seriesComboBox2">
             <property name="maximumSize">
              <size>
               <width>136</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::InsertAfterCurrent</enum>
             </property>
             <item>
              <property name="text">
               <string>#</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>##</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>###</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>####</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>#####</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>######</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>本节所有标题的级数:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>本节标题:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>本节共有多少道题:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_12">
             <property name="maximumSize">
              <size>
               <width>23</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>点</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="generatePushButton2">
          <property name="text">
           <string>生成文本并自动复制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="generateTextBrowser2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>整体升降标题</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="downPushButton">
             <property name="text">
              <string>手中标题降低1级并自动复制(Ctrl+Shift+↓)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="upPushButton">
             <property name="text">
              <string>手中标题提升1级并自动复制(Ctrl+Shift+↑)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="generateTextBrowser3"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>年份序列自动+1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_7" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>85</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="0" column="4">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>本节共有多少道题:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>本节标题:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QCheckBox" name="addSelectStrCheckBox4">
                <property name="text">
                 <string>末尾添加&quot;选择&quot;字符</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>本节所有标题的级数:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSpinBox" name="questionNumSpinBox4">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999999999</number>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="yearsSpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999999999</number>
                </property>
                <property name="value">
                 <number>2001</number>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_17">
                <property name="maximumSize">
                 <size>
                  <width>23</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>点</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QComboBox" name="seriesComboBox4">
                <property name="maximumSize">
                 <size>
                  <width>136</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertAfterCurrent</enum>
                </property>
                <item>
                 <property name="text">
                  <string>#</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>##</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>###</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>####</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>#####</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>######</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_8" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QPushButton" name="generatePushButton4">
                <property name="text">
                 <string>生成文本并自动复制</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextBrowser" name="generateTextBrowser4"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GenerateMdTitle.qrc"/>
 </resources>
 <connections/>
</ui>
